src/uu/cat/src/cat.rs

```diff
@@ -231,6 +232,8 @@ pub fn uu_app() -> Command {
         .override_usage(format_usage(USAGE))
         .about(ABOUT)
         .infer_long_args(true)
+        // Note: This is correct *only* because no argument has ArgAction::Append.
```

Comment:
Could you explain this? Because as far as I know Append takes priority over overriding.

See for instance: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=f6d36d15e74698bc254373a3d6fea383